app/views/posts/index.html.erb

```diff
@@ -1 +1,19 @@
-<p>Here is a list of posts for a given user</p>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>User's posts</title>
+</head>
+<body>
+ <div>
+  <%= render partial: "shared/user", locals: { user: @user } %>
+
+  <% @posts.each do |post| %>
+    <%= render partial: "shared/post", locals: { post: post } %>
+
+    <% post.recent_comments(post.id) do |comment| %>
+       <p>Comment # <%= comment.text %></p>
+    <% end %>
```

Comment:
- [ ] The 5 most recent comments for the posts are currently not being rendered. Please take a look at at the attached screenshot below and do your best to fix it 👎 
## Wireframe
![image](https://user-images.githubusercontent.com/46839548/189395965-25ae0fec-1361-4c63-ae36-897b66371a17.png)

## Your Implementation
![image](https://user-images.githubusercontent.com/46839548/189395812-05104729-4062-4bac-94c2-e3f3b7fcd86f.png)